packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Basic.Example.tsx

```diff
@@ -63,6 +66,20 @@ export class FacepileBasicExample extends React.Component<{}, IFacepileBasicExam
             ]}
             onChange={this._onChangePersonaSize}
           />
+          <Dropdown
+            label="Persona Presence:"
+            selectedKey={this.state.personaPresence}
+            options={[
+              { key: PersonaPresence.none, text: PersonaPresence[PersonaPresence.none] },
```

Comment:
I'm not sure it's really necessary to have the dropdown to switch presence--seems more to the point to either choose one presence and use it for all the faces, or set the presence for each face individually (which would be more realistic).

Reply:
Probably a bit overkill, but doesnt hurt. I am happy with either solution

Reply:
Alright I just dropped a little randomizer function into the example file in order to change the presences around but keep them consistent to something (the name in this case). Im assuming the user would be passing/controlling what presence the coins are via the data field of the personas array but i didn't want to change that pice of code for this.